apps/judicial-system/digital-mailbox-api/src/app/modules/cases/case.service.ts

```diff
@@ -95,6 +96,16 @@ export class CaseService {
     lang?: string,
   ): Promise<CaseResponse> {
     const response = await this.fetchCase(id, nationalId)
+    const defendant = response.defendants[0]
```

Comment:
Please find the defendant that matches the given national id in case there's more than one defendant in the case